deliver/lib/deliver/runner.rb

```diff
@@ -78,7 +78,7 @@ def upload_binary
         )
       end
 
-      transporter = FastlaneCore::ItunesTransporter.new(options[:username])
+      transporter = FastlaneCore::ItunesTransporter.new(options[:username], nil, false, options[:itc_provider])
```

Comment:
because we need to pass a value for the 4th param, we now need to provide (the default) values for the 2nd and 3rd optional params.